packages/react-native-sdk/src/utils/StreamVideoRN/types.ts

```diff
@@ -154,6 +155,11 @@ export type StreamVideoConfig = {
         title: string;
         body: string;
       };
+      /**
+       * The task to run in the foreground service
+       * The task must resolve a promise once complete
+       */
+      taskToRun: (call: Call) => Promise<void>;
```

Comment:
Hey @santhoshvai, this one should be made optional. It is a breaking change otherwise

Reply:
handled by `deepMerge`.